app/src/main/java/ru/yandex/practicum/contacts/presentation/base/BaseListDiffCallback.java

```diff
@@ -0,0 +1,17 @@
+package ru.yandex.practicum.contacts.presentation.base;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.DiffUtil;
+
+public class BaseListDiffCallback<T extends ListDiffInterface> extends DiffUtil.ItemCallback<T>{
+
+    @Override
+    public boolean areItemsTheSame(@NonNull T t, @NonNull T t1) {
+        return t.theSameAs(t1);
+    }
+
+    @Override
+    public boolean areContentsTheSame(@NonNull T t, @NonNull T t1) {
+        return false;
```

Comment:
```suggestion
        return t.equals(t1);
```
В этом методе необходимо сравнить объекты